### DISCUSSION TITLE
Flet app source code

### DISCUSSION THREAD

**Discussion by carlab23** [flet-dev/flet#3106] [General]
How can I see the source code of a Flet MacOS app (coded in Python)?

**Comment by alvaro2203** [comment]
Can you give more information? 
If the author has published the repository on github you can see in it

**Comment by baseplate-admin** [comment]
Yes and no

Short answer : NO

Long answer:
Yes in the sense that you are good with de-compiling stuffs. `flet` uses pyinstaller to convert to binary format, pyinstaller is a glorified [zipapp](https://docs.python.org/3/library/zipapp.html) . If you can decompile python bytecodes into original files
